src/pages/cli/project/override.mdx

```diff
@@ -33,4 +33,40 @@ You can override the following project-level resources that Amplify generates:
 |Amplify-generated resource|Description|
 |-|-|
 |[authRole](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)|The IAM role for authenticated access to your app backend|
-|[unauthRole](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)|The IAM role for authenticated or guest access to your app backend|
\ No newline at end of file
+|[unauthRole](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html)|The IAM role for authenticated or guest access to your app backend|
+
+
+## Customize unauthRole policies
+
+For example, to use the project override to update the unauthRole policy for S3, use the following:
```

Comment:
Can we pick another example. Other than S3? Especially because customers can configure this via `amplify update storage` without an override.

Reply:
updated with geo example